### PR TITLE
helm push to artifactory

### PR DESCRIPTION
**Pull request by scottchc1**
**Description of the change**
internal jenkins job to push helm charts to artifactory

**Benefits**
compliance

**Drawbacks**
none

**Applicable issues**
none

**Additional information**

**Checklist** 
- [] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [] Variables are documented in the README.md
- [] Title of the PR starts with chart name (e.g. `[charts/gateway]`)
- [] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

### CODE REVIEW

**Review by Gazza7205 — APPROVED** [review]
Looks good to me

File: push_helm_charts.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            for chunk in r.iter_content(chunk_size=32768):
+                f.write(chunk)
+    print(f"downloaded {local_filename}")
+    return local_filename
+
+def main():
+    path = Path(args.index)
+    yaml = YAML(typ='safe')
+    data = yaml.load(path)
+    druid_url = data["entries"]["druid"][0]["urls"][0]
```
Comment (jmmAVGO) [inline push_helm_charts.py:49]: These and the trios of three of calls below all seem so similar, can you make them into a function?

**Review by jmmAVGO — COMMENTED** [review]
Seems mostly fine, noted opportunity for reusable function

File: push_helm_charts.py (hunk 1 above)
Comment (scottchc1) [reply push_helm_charts.py:49]: download chart is already a function. don't think other parts are complex enough

**Pull request merged**